src/taskclass.js

```diff
@@ -0,0 +1,25 @@
+export default class Task {
+  constructor(taskEntry) {
+    this.taskEntry = taskEntry;
+    this.completed = false;
+    this.index = 0;
+  }
+
+  addTask(taskList) {
+    const myTask = new Task(this.taskEntry);
+    myTask.index = taskList.length;
+    taskList.push(myTask);
+    localStorage.setItem('taskDetail', JSON.stringify(taskList));
+  }
+
+  static delTask(index, taskList) {
+    taskList = taskList.filter((tasks, i) => index !== i);
+    localStorage.setItem('taskDetail', JSON.stringify(taskList));
+    window.location.reload();
+  }
```

Comment:
- [ ] Following the project requirements, deleting a task should update all remaining tasks' indexes, so they represent the current list order and are **unique**. As you see on the screenshot, I deleted `task 2`, I added `task 4` and I noticed that `task 4` and `task 3` have the same index. Kindly work on that.

<img width="949" alt="image" src="https://user-images.githubusercontent.com/77190203/161300829-d4f36508-f34b-402d-8ccb-a38b8d1d2e11.png">